spec/features/workflow/schedule_set_datetime_passes_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+RSpec.feature "Set scheduled publishing date and time passes before scheduling" do
+  scenario do
+    given_there_is_an_edition_with_a_scheduled_publishing_date_and_time_set_before_now
+    when_i_visit_the_summary_page
+    then_i_cannot_see_a_schedule_link
+  end
+
+  def given_there_is_an_edition_with_a_scheduled_publishing_date_and_time_set_before_now
+    @edition = create(:edition)
+    user = User.first
+    datetime = Time.zone.now - 1
+    revision = @edition.revision.build_revision_update(
+      { scheduled_publishing_datetime: datetime }, user
+    )
+    @edition.assign_revision(revision, user).save!
```

Comment:
We can hopefully do all of this through the factory by: `create(:edition, scheduled_publishing_datetime: Time.zone.now - 1)`

It may need the factory updating it lacks that attribute.

Reply:
I've updated the factories to handle `scheduled_publishing_datetime` 👍